docker-compose.prod.yml

```diff
@@ -2,7 +2,6 @@ version: '2.3'
 services:
   web:
     image: 997386117669.dkr.ecr.us-east-1.amazonaws.com/impact-api
-    env_file: .prod.env
```

Comment:
I assume this change has nothing to do with the main PR, so give me some context on this change and I'll review it as well

Reply:
adding this env file as part of the production image that gets pushed by travis causes anything defined in the env file to be un-overridable by the task configuration which is undesirable.